result_scripts/Data-Analysis.py

```diff
@@ -0,0 +1,62 @@
+

```

Comment:
What does this file do? Is it your preparation script?

If yes I suggest not to upload it, bz this will confuse the teacher that we have several preparation scripts